src/Extension/FluentGridFieldExtension.php

```diff
@@ -23,19 +21,6 @@
 class FluentGridFieldExtension extends Extension
 {
     use FluentAdminTrait;
-    use FluentBadgeTrait;
-
-    /**
-     * Push a badge to indicate the language that owns the current item
-     *
-     * @param DBField|null $badgeField
-     * @see VersionedGridFieldItemRequest::Breadcrumbs()
-     */
-    protected function updateBadge(&$badgeField)
```

Comment:
That hook doesn't exist anymore - this is now handled through generalised status flag rendering in core.